centrallix/wgtr/wgtr.c

```diff
@@ -1515,7 +1518,8 @@ wgtrSetProperty(pWgtrNode widget, char* name, int datatype, pObjData val)
 pWgtrNode 
 wgtrNewNode(	char* name, char* type, pObjSession s,
 		int rx, int ry, int rwidth, int rheight,
-		int flx, int fly, int flwidth, int flheight)
+		int flx, int fly, int flwidth, int flheight,
```

Comment:
So we modify the function signature for wgtrNewNode() here, but this branch does not appear to modify any of the locations where the function is called. How do you intend for these new values to be generated?